driver/fixtures/fixtures_test.go

```diff
@@ -0,0 +1,52 @@
+package fixtures
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/bblfsh/bash-driver/driver/normalizer"
+
+	"gopkg.in/bblfsh/sdk.v2/driver"
+	"gopkg.in/bblfsh/sdk.v2/driver/fixtures"
+	"gopkg.in/bblfsh/sdk.v2/driver/native"
+)
+
+const projectRoot = "../../"
+
+var Suite = &fixtures.Suite{
+	Lang: "bash",
+	Ext:  ".bash",
```

Comment:
`.sh`?

Reply:
I prefer `.bash` for the fixtures since sh could be understood as POSIX-sh and some tests use bash features.